src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -65,18 +66,18 @@ object VariantSampleMatrix {
 
   def genGeneric(hc: HailContext): Gen[VariantSampleMatrix[Annotation]] =
     for (tSig <- Type.genArb.resize(3);
-      vsm <- VSMSubgen[Annotation](
```

Comment:
Here's mine:
```
<code_scheme name="Default (1)">
  <ScalaCodeStyleSettings>
    <option name="SPACES_IN_ONE_LINE_BLOCKS" value="true" />
  </ScalaCodeStyleSettings>
  <XML>
    <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
  </XML>
  <codeStyleSettings language="Scala">
    <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
    <option name="KEEP_SIMPLE_BLOCKS_IN_ONE_LINE" value="true" />
    <option name="KEEP_SIMPLE_METHODS_IN_ONE_LINE" value="true" />
  </codeStyleSettings>
</code_scheme>
```
Here is Hail's:
```
<code_scheme name="Hail">
  <MarkdownNavigatorCodeStyleSettings>
    <option name="RIGHT_MARGIN" value="72" />
  </MarkdownNavigatorCodeStyleSettings>
  <ScalaCodeStyleSettings>
    <option name="NOT_CONTINUATION_INDENT_FOR_PARAMS" value="true" />
    <option name="SPACES_IN_ONE_LINE_BLOCKS" value="true" />
    <option name="ENABLE_SCALADOC_FORMATTING" value="false" />
    <option name="ENFORCE_FUNCTIONAL_SYNTAX_FOR_UNIT" value="false" />
  </ScalaCodeStyleSettings>
  <XML>
    <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
  </XML>
  <codeStyleSettings language="Scala">
    <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
    <option name="ALIGN_MULTILINE_FOR" value="false" />
```
Strangely, Hails has another 600 repetitive lines.